src/libostree/ostree-repo.c

```diff
@@ -4598,6 +4602,9 @@ ostree_repo_regenerate_summary (OstreeRepo     *self,
       const char *commit = g_hash_table_lookup (refs, ref);
       g_autofree char *remotename = NULL;
       g_autoptr(GVariant) commit_obj = NULL;
+      g_auto(GVariantDict) commit_metadata_builder = OT_VARIANT_BUILDER_INITIALIZER;
```

Comment:
You probably could use `G_VARIANT_DICT_INIT (NULL)` here, unless the GLib version requirement is too low.